.github/workflows/test.yml

```diff
@@ -77,14 +77,6 @@ jobs:
     - name: Test js library
       run: sudo -u testbot bash -lc 'CI=true make test-web'
 
-    - if: failure()
-      run: |
-        pwd
-        ls rpc
-        ls rpc/examples
-        ls rpc/examples/echo/
-        ls rpc/examples/echo/playwright-report
```

Comment:
[q] Just didn't need this extra debug information?

Reply:
yeah, this was testing from another PR - should have been removed there